include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -206,15 +206,7 @@ class logging_resource_adaptor final : public device_memory_resource {
   void* do_allocate(std::size_t bytes, cuda_stream_view stream) override
   {
     auto const p = upstream_->allocate(bytes, stream);
-    std::string msg{"allocate,"};
-    std::stringstream ss;
-    ss << p;
-    msg += ss.str();
-    msg += ",";
-    msg += std::to_string(bytes);
-    msg += ",";
-    msg += std::to_string(uintptr_t{stream});
-    logger_->info(msg);
+    logger_->info("allocate,{},{},{}", p, bytes, fmt::ptr(stream.value()));
```

Comment:
I'd be curious what the overhead of this is compared to what I had before. The reason I had it the way it was before is because I benchmarked it and found it was the fastest way to do it.

Reply:
This is surprising from a library called "speed log".

Reply:
I never tried the method used here, so it could be faster/just as fast. My test was testing using string concat vs a sstream and other related methods.

Reply:
I think spdlog is heavily invested in this libfmt approach, so I hope it is fast. If it becomes a problem we can switch back.  I guess we don't have a logging benchmark, might be good to add.